daemon/networkdriver/bridge/driver.go

```diff
@@ -302,6 +302,11 @@ func setupIPTables(addr net.Addr, icc, ipmasq bool) error {
 	)
 
 	if !icc {
+		output, err := ioutil.ReadFile("/proc/sys/net/bridge/bridge-nf-call-iptables")
+		if err != nil || string(output[:1]) == "0" {
+			log.Warnf("net.bridge.bridge-nf-call-iptables is not enabled. So --icc=false does not work.")
```

Comment:
Thanks! As mentioned below, I'm not a maintainer, so I'll leave it to them to decide if this if the desired approach or a hard "error" is more appropriate :)